demo/radio-button-styling-demos.html

```diff
@@ -0,0 +1,54 @@
+<dom-module id="radio-button-styling-demos">
+  <template>
+    <style include="vaadin-component-demo-shared-styles">
+        :host {
+          display: block;
+        }
+      </style>
+    <h3>Customizing font</h3>
+    <vaadin-demo-snippet id="customizing-font" when-defined="vaadin-radio-button">
+      <template preserve-content>
+        <style>
+              vaadin-radio-button {
+                font-weight: bold;
+                font-family: sans-serif;
+              }
```

Comment:
The CSS is not properly indented. Please use 2 spaces.

Reply:
Done.